rpc/jsonrpc/client/ws_client.go

```diff
@@ -18,12 +18,23 @@ import (
 	types "github.com/tendermint/tendermint/rpc/jsonrpc/types"
 )
 
-const (
-	defaultMaxReconnectAttempts = 25
-	defaultWriteWait            = 0
-	defaultReadWait             = 0
-	defaultPingPeriod           = 0
-)
+// WSOptions for WSClient.
+type WSOptions struct {
+	MaxReconnectAttempts uint          // maximum attempts to reconnect
+	ReadWait             time.Duration // deadline for any read op
+	WriteWait            time.Duration // deadline for any write op
+	PingPeriod           time.Duration // frequency with which pings are sent
+}
+
+// DefaultWSOptions returns default WS options.
+func DefaultWSOptions() WSOptions {
+	return WSOptions{
+		MaxReconnectAttempts: 25,
```

Comment:
Lol 25 is way too many reconnect attempts for exponential backoff. This would mean that the last attempt happens more than a year later (2^25)

Reply:
Maybe 10 is enough

Reply:
yep, good catch!